docs/deploy/upgrading.mdx

```diff
@@ -16,22 +16,19 @@ received by updating one or more of the following extensions:
 To inspect the current version of an extension, run the following command.
 
 ```sql
-SELECT extversion FROM pg_extension WHERE extname = '<extension_name>';
+-- Get the version of pg_search
+SELECT extversion FROM pg_extension WHERE extname = 'pg_search';
+-- Get the version of pg_analytics
+SELECT extversion FROM pg_extension WHERE extname = 'pg_analytics';
 ```
 
-<Accordion title="Example Usage">
-  ```sql
-  SELECT extversion FROM pg_extension WHERE extname = 'pg_search';
-  ```
-</Accordion>
-
 ## Getting the Latest Version
 
 Because `pg_search`, `pg_analytics`, and `pgvector` are independent extensions, they each have their own versions.
 For the latest available version, please refer to the respective Github repos:
 
 1. [`pg_search`](https://github.com/paradedb/paradedb/releases) is on version `0.9.1`
-2. [`pg_analytics`](https://github.com/paradedb/pg_analytics/blob/main/pg_analytics.control) is on version `0.1.0`
+2. [`pg_analytics`](https://github.com/paradedb/pg_ingest/blob/main/pg_analytics.control) is on version `0.1.0`
```

Comment:
is this intentional? I thought we were keeping analytics. I think we should keep the pg_analytics name for at least a bit longer, until we're sure of this